src/main/java/build/buildfarm/server/services/OperationQueueService.java

```diff
@@ -40,8 +40,6 @@ public OperationQueueService(Instance instance) {
   private static class OperationQueueMatchListener implements MatchListener {
     private final InterruptingPredicate<QueueEntry> onMatch;
 
-    private static final QueueEntry queueEntry = null;
```

Comment:
made static because of a once `FinalFieldCouldBeStatic`, I guess I'm just glad there's a route to removing these now...